components/datasets/form/steps/Step1.js

```diff
@@ -57,6 +64,42 @@ class Step1 extends React.Component {
     this.props.onChange({ legend });
   }
 
+  onSubscribableChange(obj) {
+    const { subscribable } = this.props.form;
```

Comment:
```
    const subscribable = this.props.form.subscribable.slice(0);
    const index = subscribable.findIndex(s => s.id === obj.id);

    subscribable[index] = {
      ...subscribable[index],
      ...obj
    };

    this.props.onChange({ subscribable });

```

Reply:
This function is throwing an error because you are using the id as an index. What happens if I create an element, then create another one but I decide to remove the first one. ERROR if I try to edit the second I created. Try to use the function above

Reply:
👍